addon/index.js

```diff
@@ -1,7 +1,2 @@
-import { modifier } from 'ember-modifier';
-
-export default modifier(function prop(element, positional, named) {
-  for (let key in named) {
-    element[key] = named[key];
-  }
-});
+import prop from './modifiers/prop';
+export default prop;
```

Comment:
I forgot about this before, but you should actually be able to do this:

```suggestion
export { default } from './modifiers/prop';
```